pages/accountLists/[accountListId]/settings/preferences.page.tsx

```diff
@@ -29,17 +30,38 @@ import { useAccountListId } from 'src/hooks/useAccountListId';
 import { useGetTimezones } from 'src/hooks/useGetTimezones';
 import { getCountries } from 'src/lib/data/countries';
 import { suggestArticles } from 'src/lib/helpScout';
+import theme from 'src/theme';
 import { SettingsWrapper } from './Wrapper';
 
 const AccordionLoading = styled(Skeleton)(() => ({
   width: '100%',
   height: '48px',
 }));
 
+const StickyBox = styled(Box)(() => ({
```

Comment:
You can get the theme here instead of importing it.

```suggestion
const StickyBox = styled(Box)(({ theme }) => ({
```